libs/dh/shared/data-access-api/project.json

```diff
@@ -23,5 +23,5 @@
       }
     }
   },
-  "tags": ["product:dh", "scope:shared", "type:data-access", "type:open-api"]
+  "tags": ["product:dh", "scope:shared", "type:data-access"]
```

Comment:
suggestion: If we don't need the `"type:open-api"` tag anymore, there are some files where references to `open-api` need to be removed. Like `.eslintrc.json` and `README.md` on root.